common/reviews/api/api-extractor-model.api.md

```diff
@@ -698,9 +723,7 @@ export interface IApiCallSignatureOptions extends IApiTypeParameterListMixinOpti
 }
 
 // @public
-export interface IApiClassOptions extends IApiItemContainerMixinOptions, IApiNameMixinOptions, IApiReleaseTagMixinOptions, IApiDeclaredItemOptions, IApiTypeParameterListMixinOptions {
-    // (undocumented)
-    extendsTokenRange: IExcerptTokenRange | undefined;
```

Comment:
I didn't realize that `ApiClass.extendsTokenRange` got renamed to `extendsTokenRanges`.  Per our chat, let's see if we can avoid that because it would be a breaking change for consumers, without a clear benefit for consumers.

Changing to an array in the .api.json file format is fine though, as long as we bump the `ApiJsonSchemaVersion`.

Reply:
Updated this PR to avoid that. The `.api.json` format is no longer changing as well.